### ISSUE TITLE
Remove PSR2.Namespaces.UseDeclaration.MultipleDeclarations

### ISSUE THREAD

**Issue by paulgibbs** [humanmade/coding-standards#24]
Can we remove PSR2.Namespaces.UseDeclaration.MultipleDeclarations ("There must be one USE keyword per declaration")?

Modern PHP supports this syntax, and the above rule blocks it:

```
use some\namespace\{ClassA, ClassB, ClassC as C};
use function some\namespace\{fn_a, fn_b, fn_c};
use const some\namespace\{ConstA, ConstB, ConstC};
```

**Comment by rmccue** [comment]
Missed this issue when it came in.

I think we can do this, just need to make sure that the multiple keywords are also alphabetised.

**Comment by paulgibbs** [comment]
I just ran into this again.

**Comment by nathanielks** [comment]
@paulgibbs sounds like an opportunity to submit a PR!

**Comment by tfrommen** [comment]
Included in 5d3de4f341815c62b0f7bcce477738ba3c77b074.
